backend/requirements.txt

```diff
@@ -23,3 +23,5 @@ toml==0.10.2
 typed-ast==1.4.1
 typing-extensions==3.7.4.3
 Werkzeug==1.0.1
+#added psycopg2-binary to run the test script
+psycopg2-binary==2.8.6
```

Comment:
@peterghrong I was wondering whether you've tried it with just psycopg2-2.8.6? The issue you had before maybe caused by psycopg2-2.7.3 and I have since updated it back to 2.8.6. You can test it by removing your virtual environment directory and installing the dependencies again.

Reply:
@peterghrong jk keeping both should capture everyone's situation. Mb!